src/js/pensions/helpers.jsx

```diff
@@ -61,7 +61,7 @@ export function fileHelp({ formContext }) {
   return (
     <p>
       Files we accept: pdf, jpg, png<br/>
-      Maximum file size: 2MB
+      Maximum file size: 20MB
```

Comment:
Yeah, I was just pondering through it and I couldn't come up with a good answer. We'll just have to be careful to make sure they match is all.